iroh-net/src/lib.rs

```diff
@@ -31,6 +31,17 @@ pub mod util;
 pub use magic_endpoint::{AddrInfo, MagicEndpoint, NodeAddr};
 
 pub use iroh_base::key;
+
+/// The identifier for a node in the (iroh) network.
+///
+/// Each [`MagicEndpoint`] is uniquely identified by this.
+///
+/// This is equivalent to [`iroh_base::key::PublicKey`].  By convention we will (or should)
+/// use `PublicKey` as type name when performing cryptographic operations, but use `NodeId`
+/// when referencing a node.  E.g.:
+///
+/// - `encrypt(key: PublicKey)`
+/// - `send_to(node: NodeId)`
 pub use iroh_base::key::PublicKey as NodeId;
```

Comment:
Can you move the type alias, including the docs, to `iroh_base` and pub use it here?

It was an oversight on my side in # to not move the type alias as well.

Reply:
done